packages/xchain-bitcoin/src/const.ts

```
@@ -128,6 +129,26 @@ export const BitgoProviders: UtxoOnlineDataProviders = {
   [Network.Mainnet]: mainnetBitgoProvider,
 }
 
+// NowNodes Blockbook data providers for Bitcoin
+// Docs: https://nownodes.io/nodes/bitcoin-btc
+const mainnetBtcBlockbookProvider = new BlockbookProvider(
+  'https://btcbook.nownodes.io/api/v2',
+  AssetBTC,
+  BTC_DECIMAL,
```

ESLint

[error] Replace `⏎··'https://btcbook.nownodes.io/api/v2',⏎··AssetBTC,⏎··BTC_DECIMAL,` with `'https://btcbook.nownodes.io/api/v2',·AssetBTC,·BTC_DECIMAL,·{`

(prettier/prettier)